deploy/manifests/deployment.yaml

```diff
@@ -20,15 +20,15 @@ spec:
     spec:
       containers:
       - args:
-        - --cloud-config=/etc/kubernetes/cloud-config/cloud-config.toml
+        - --cloud-config=/etc/kubernetes/cloud-config
```

Comment:
Should we rename the flag as `--kubeconfig` given that it is a kubeconfig file now?

Reply:
No. `cloud-config` is a flag defined by Kubernetes core. The config will be loaded into a reader which can be read when registering a new cloud controller manager.